diff --git a/node_modules/@backstage/plugin-scaffolder-backend/dist/cjs/ScaffolderEntitiesProcessor-021c5dcb.cjs.js b/node_modules/@backstage/plugin-scaffolder-backend/dist/cjs/ScaffolderEntitiesProcessor-021c5dcb.cjs.js
index 2b93a75..87bbcb8 100644
--- a/node_modules/@backstage/plugin-scaffolder-backend/dist/cjs/ScaffolderEntitiesProcessor-021c5dcb.cjs.js
+++ b/node_modules/@backstage/plugin-scaffolder-backend/dist/cjs/ScaffolderEntitiesProcessor-021c5dcb.cjs.js
@@ -42,6 +42,7 @@ var pluginPermissionCommon = require('@backstage/plugin-permission-common');
 var url = require('url');
 var os = require('os');
 var pluginCatalogNode = require('@backstage/plugin-catalog-node');
+var jose = require('jose');
 
 function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e : { 'default': e }; }
 
@@ -3555,7 +3556,6 @@ function createPublishGithubAction(options) {
         squashMergeCommitMessage = "COMMIT_MESSAGES",
         allowRebaseMerge = true,
         allowAutoMerge = false,
-        collaborators,
         hasProjects = void 0,
         hasWiki = void 0,
         hasIssues = void 0,
@@ -3563,6 +3563,20 @@ function createPublishGithubAction(options) {
         token: providedToken,
         requiredCommitSigning = false
       } = ctx.input;
+      let collaborators = ctx.input.collaborators;
+      // If collaborators not added from template.. below script would add current logged in github userId as collaborator with admin role
+      if (!collaborators) {
+        const payloadUser = ctx.secrets?.backstageToken ? await jose.decodeJwt(ctx.secrets.backstageToken) : null;
+        let loggedInUserId = payloadUser?.sub?.split('/')[1];
+        if (payloadUser && loggedInUserId)
+          collaborators = [
+            {
+              user: loggedInUserId,
+              access: 'admin'
+            }
+          ];
          console.log("i m in collaborators----",collaborators,payloadUser,loggedInUserId);
+      }
+      // End default collaborator script
       const octokitOptions = await getOctokitOptions({
         integrations,
         credentialsProvider: githubCredentialsProvider,
