diff --git a/node_modules/@backstage/plugin-github-actions/dist/index.esm.js b/node_modules/@backstage/plugin-github-actions/dist/index.esm.js
index b37e4e5..0fc8a0e 100644
--- a/node_modules/@backstage/plugin-github-actions/dist/index.esm.js
+++ b/node_modules/@backstage/plugin-github-actions/dist/index.esm.js
@@ -27,7 +27,7 @@ class GithubActionsClient {
   }
   async getOctokit(hostname) {
     var _a;
-    const token = await this.githubAuthApi.getAccessToken(["repo"]);
+    const token = await this.githubAuthApi.getAccessToken(["read:user"]);
     const configs = readGithubIntegrationConfigs(
       (_a = this.configApi.getOptionalConfigArray("integrations.github")) != null ? _a : []
     );
