diff --git a/node_modules/@backstage-community/plugin-announcements-backend/dist/service/persistence/AnnouncementsDatabase.cjs.js b/node_modules/@backstage-community/plugin-announcements-backend/dist/service/persistence/AnnouncementsDatabase.cjs.js
index be1ee5c..fcc40bc 100644
--- a/node_modules/@backstage-community/plugin-announcements-backend/dist/service/persistence/AnnouncementsDatabase.cjs.js
+++ b/node_modules/@backstage-community/plugin-announcements-backend/dist/service/persistence/AnnouncementsDatabase.cjs.js
@@ -34,7 +34,7 @@ const parseTagsFromDb = (tags) => {
 const announcementUpsertToDB = (announcement) => {
   return {
     id: announcement.id,
-    category: announcement.category ? slugify__default.default(announcement.category, {
+    category: announcement.category ? slugify__default.default(typeof announcement.category === 'object' ? announcement.category.slug : announcement.category, {
       lower: true
     }) : announcement.category,
     title: announcement.title,
